platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.platform

```diff
@@ -980,6 +980,27 @@ bus named-singleton GPIO {
 	signal digitalOut(pin : GPIOPin, initialValue : bool = false) : bool
 }
 
+export exception EndOfFileException
+
+io named-singleton SDCard {
+	generator "org.eclipse.mita.platform.xdk110.io.SDCardGenerator"
+	sizeInferrer "org.eclipse.mita.platform.xdk110.io.SDCardSizeInferrer"
+
+	signal resumingFile_Text_Read(filePath: string, blockSize: uint32) : string
```

Comment:
We do not have a proper convention yet for signal names, but I am not a fan of `CamelCase_mixed_With_PolishNotation`. I think following naming scheme looks way nicer and is more in line with what people (coming from e.g. Typescript) would be used to:

- `resumingTextRead`
- `appendingTextWrite`
- `rewindingTextRead`
- ...